contain_learning/Dockerfile

```
@@ -0,0 +1,5 @@
+FROM centos:latest
```

Identified a potential vulnerability, here’s a suggested change for The base image 'centos:latest' can lead to non-deterministic builds as it may change over time. It's recommended to use a specific version tag.:

```suggestion
FROM centos:7
```